components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2161,13 +2161,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             // NOTE: this is a temporary workaround, as long as we're not automatically re-create the customer
             // entity on Stripe to support a switch of currencies, we're taking an exit here.
             if (customer.currency !== currency) {
-                throw new ResponseError(
-                    ErrorCodes.SUBSCRIPTION_ERROR,
-                    `Your previous subscription was in ${
-                        customer.currency || "unknown"
-                    }. If you'd like to change currencies, please contact our support.`,
-                    { hint: "currency", oldValue: customer.currency, value: currency },
-                );
+                log.error(`The previous subscription was in ${customer.currency || "unknown"}.`, {
```

Comment:
If we're not stopping here, the subsequent `billingService.createStripeCustomer` will fail if there is a diff in currency.

Reply:
ok

Reply:
Also, it would be interesting to see why `billingService.getStripeCustomer` does not carry the previous currency in such case.

Recently we switched to manage this value in DB instead of fetching this from Stripe. It would be great to check if such customer entries in DB do contain the currency values. If not, we'll face the issue mentioned in the previous comment.

Reply:
we forgot to apply a default for the already created customers.